File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/SuspensionDto.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

public class SuspensionDto {

    private Long beginMoment;
    private Long endMoment;
    private String suspensionReason;

    public Long getBeginMoment() { return beginMoment; }

    public void setBeginMoment(Long beginMoment) { this.beginMoment = beginMoment; }

    public Long getEndMoment() { return endMoment; }

    public void setEndMoment(Long endMoment) { this.endMoment = endMoment; }

    public String getSuspensionReason() { return suspensionReason; }

    public void setSuspensionReason(String suspensionReason) { this.suspensionReason = suspensionReason; }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/SuspensionDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import com.github.pozo.KotlinBuilder
import java.io.Serializable

@KotlinBuilder
data class SuspensionDto(
        val beginMoment: Long? = null,
        val endMoment: Long? = null,
        val suspensionReason: String? = null
) : Serializable
